drivers/clock_control/clock_control_mcux_syscon.c

```diff
@@ -80,7 +81,24 @@ static int mcux_lpc_syscon_clock_control_get_subsys_rate(
 		*rate = CLOCK_GetSdioClkFreq(1);
 		break;
 #endif
+#if defined(CONFIG_COUNTER_MCUX_CTIMER)
+	case (MCUX_CTIMER0_CLK + MCUX_CTIMER_CLK_OFFSET):
+		*rate = CLOCK_GetCTimerClkFreq(0);
+		break;
+	case (MCUX_CTIMER1_CLK + MCUX_CTIMER_CLK_OFFSET):
+		*rate = CLOCK_GetCTimerClkFreq(1);
+		break;
+	case (MCUX_CTIMER2_CLK + MCUX_CTIMER_CLK_OFFSET):
+		*rate = CLOCK_GetCTimerClkFreq(2);
+		break;
+	case (MCUX_CTIMER3_CLK + MCUX_CTIMER_CLK_OFFSET):
+		*rate = CLOCK_GetCTimerClkFreq(3);
+		break;
+	case (MCUX_CTIMER4_CLK + MCUX_CTIMER_CLK_OFFSET):
+		*rate = CLOCK_GetCTimerClkFreq(4);
+		break;
 	}
+#endif
```

Comment:
I think this `#endif` should move after line 99 so the closing brace for the switch command is available for all test cases.

Reply:
Didn't spot that. Fixed it now.